Tasks/JenkinsQueueJob/jenkinsqueuejobtask.ts

```diff
@@ -22,155 +22,852 @@ var username = serverEndpointAuth['parameters']['username'];
 var password = serverEndpointAuth['parameters']['password'];
 
 var jobName = tl.getInput('jobName', true);
-var jobQueueUrl = serverEndpointUrl + '/job/' + jobName + '/build';
-tl.debug('jobQueueUrl=' + jobQueueUrl);
 
 var captureConsole = tl.getBoolInput('captureConsole', true);
-var captureConsolePollInterval = 5000; // five seconds is what the Jenkins Web UI uses
+var pollInterval = 5000; // five seconds is what the Jenkins Web UI uses
+
+// capturePipeline is only possible if captureConsole mode is enabled
+var capturePipeline = captureConsole ? tl.getBoolInput('capturePipeline', true) : false;
+
+var parameterizedJob = tl.getBoolInput('parameterizedJob', true);
+
+var jobQueueUrl = addUrlSegment(serverEndpointUrl, '/job/' + jobName);
+jobQueueUrl += (parameterizedJob) ? '/buildWithParameters?delay=0sec' : '/build?delay=0sec';
+tl.debug('jobQueueUrl=' + jobQueueUrl);
+
+function addUrlSegment(baseUrl: string, segment: string): string {
```

Comment:
not quite the same; this method ensures exactly one / is between the base and segment